src/facades/BaseFacade.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+namespace SwitchViews\facades;
```

Comment:
It is ideal to that namespace names should start with Capital Letter. In this case **facades** should be **Facades**